src/scripts/build_all_repos.sh

```diff
@@ -908,6 +911,23 @@ EOF
 
 }
 
+function test_rpms() {
+
+    # Not the fastest way ever
+    for norepl in $NOREPLACE_FILES; do
+        for rpm in `ls $RPMS/*rpm`; do
```

Comment:
I think this is a little bit an overkill to search all RPM for everyfile. Why not giving a list of RPM/file? For example:
```bash
NOREPLACE_FILES="ccm:/etc/ccm.conf cdp-listend:/etc/cdp-listend.conf ncm:/etc/ncm-ncd.conf ncm-cdispd:/etc/ncm-cdispd.conf"
```

And then parse the value to separate the RPM name and the file name.